cmd/osbuild-worker/main.go

```diff
@@ -90,12 +92,13 @@ func main() {
 		Azure *struct {
 			Credentials string `toml:"credentials"`
 		} `toml:"azure"`
+		OfflineToken string `toml:"offline_token"`
```

Comment:
Should this be an external file, like the other credentials? I think we might print this file to the logs.

Reply:
ahh yea, good point, as this is a secret